store/extensions/CDHelper/extension.json

```
@@ -0,0 +1,39 @@
+{
+    "title": "CD Helper",
+    "description": "A handy G-Earth extension that adds useful tools to manage your CDs in-game!",
+    "authors": [
+      {
+        "name": "jonny7k",
+        "hotel": "com.br",      
+        "discord": "jonny_7k",
+
+      }
+    ],
+    "version": "0.1",
+    "categories": ["Usability", "Others"],
+    
+    "source": "https://github.com/jonnymariani/cdhelper",
+    "readme": "https://github.com/jonnymariani/cdhelper/blob/main/README.md",
+    "releases": "https://github.com/jonnymariani/cdhelper/releases"
+    
```

```suggestion
    
    "source": "https://github.com/jonnymariani/cdhelper",
    "readme": "https://github.com/jonnymariani/cdhelper/blob/main/README.md",
    
```
You don't have any releases listed on your github so either remove the line from here or list your releases there (and then add a command behind the string here)